tests/RobotFramework/tasks/E2E_Tutorials/connect_to_c8y_send_measurements.robot

```diff
@@ -0,0 +1,201 @@
+    ${output}=    Execute Command    sudo tedge connect c8y
+    Log    ${output}
+    Should Contain    ${output}    Checking if systemd is available.
+    Should Contain    ${output}    Checking if configuration for requested bridge already exists.
+    Should Contain    ${output}    Validating the bridge certificates.
+    Should Contain    ${output}    Creating the device in Cumulocity cloud.
+    Should Contain    ${output}    Saving configuration for requested bridge.
+    Should Contain    ${output}    Restarting mosquitto service.
+    Should Contain    ${output}    Awaiting mosquitto to start. This may take up to 5 seconds.
+    Should Contain    ${output}    Enabling mosquitto service on reboots.
+    Should Contain    ${output}    Successfully created bridge connection!
+    Should Contain    ${output}    Sending packets to check connection. This may take up to 2 seconds.
+    Should Contain    ${output}    Connection check is successful.
+    Should Contain    ${output}    Checking if tedge-mapper is installed.
+    Should Contain    ${output}    Starting tedge-mapper-c8y service.
+    Should Contain    ${output}    Persisting tedge-mapper-c8y on reboot.
+    Should Contain    ${output}    tedge-mapper-c8y service successfully started and enabled!
+    Should Contain    ${output}    Enabling software management.
+    Should Contain    ${output}    Checking if tedge-agent is installed.
+    Should Contain    ${output}    Starting tedge-agent service.
+    Should Contain    ${output}    Persisting tedge-agent on reboot.
+    Should Contain    ${output}    tedge-agent service successfully started and enabled!
+    Log    Connected to Cumulocity IoT and verified connection for ${DEVICE_SN}
+
+Send Temperature Measurement
+    [Documentation]    Send a temperature measurement.
+    Execute Command    tedge mqtt pub 'te/device/main///m/environment' '{"temperature": 25}'
+    Log    Sent temperature measurement: 25°C
+
+Send Three Phase Current Measurement
+    [Documentation]    Send a three-phase current measurement.
+    Execute Command    tedge mqtt pub 'te/device/main///m/electrical' '{"three_phase_current": {"L1": 9.5, "L2": 10.3, "L3": 8.8}}'
+    Log    Sent three-phase current measurement
+
+Send Combined Measurement
+    [Documentation]    Send a combined measurement.
+    Execute Command    tedge mqtt pub 'te/device/main///m/combined' '{"time": "2020-10-15T05:30:47+00:00", "temperature": 25, "current": {"L1": 9.5, "L2": 10.3, "L3": 8.8}, "pressure": 98}'
+    Log    Sent combined measurement
+
+Send Child Device Temperature Measurement
+    [Documentation]    Send a temperature measurement to a child device.
+    Execute Command    tedge mqtt pub 'te/device/child1///m/environment' '{"temperature": 25}'
+    Log    Sent temperature measurement to child device: 25°C
+
+Verify Measurement In Cumulocity
+    Device Should Have Measurements    type=environment    minimum=1    maximum=1
+
```

Comment:
It would make more sense to make this a Keyword (function) that takes an argument for the type, then you can reuse it for the other measurements.

```
Verify Measurement In Cumulocity
    [Arguments]        ${type}
    Device Should Have Measurements    type=${type}    minimum=1    maximum=1

```

Then the usage is just:

```
Verify Measurement In Cumulocity    type=environment
```